.env.example

```diff
@@ -1,2 +1,6 @@
 ALCHEMY_API_ENDPOINT=
+INFURA_KEY=
```

Comment:
I guess this one is no longer needed?

Reply:
it's needed for the ropsten hardhat config block

we probably wont need that block down the line but figured i'd add it in since you cant deploy the repo as it currently is without it